pkg/cmd/tests/tests_run.go

```diff
@@ -90,14 +60,13 @@ type RunOptions struct {
 	ParallelServerAddress string
 	ParallelLogLevel      int32
 
+	testSuites    ginkgotest.TestSuites
```

Comment:
I'd expect this to be public, so it's easily extensible as the other fields

Reply:
done